### ISSUE TITLE
Failed to clone repository: 'git@github.com:rapidsai/cumlprims_mg.git'

### ISSUE THREAD

**Issue by stelgic** [rapidsai/node#462]
..

**Comment by stelgic** [comment]
reported issue not required

**Comment by trxcllnt** [comment]
Yeah, cumlprims_mg is a closed-source repository. I forgot to add a CMake flag to disable it, but in the meantime you should be able to build by commenting out [this line](https://github.com/rapidsai/node/blob/d28a85e10ac9e01e10a27468bbc866320facf241/modules/core/cmake/Modules/ConfigureCUML.cmake#L23) and setting [this option](https://github.com/rapidsai/node/blob/d28a85e10ac9e01e10a27468bbc866320facf241/modules/core/cmake/Modules/ConfigureCUML.cmake#L55) to `OFF`.
